projects/core/src/alert/alert.element.spec.ts

```diff
@@ -53,6 +53,22 @@ describe('Alert element – ', () => {
       await componentIsStable(component);
       expect(component.querySelectorAll('cds-internal-close-button').length).toBe(0);
     });
+
+    it('should not let content overflow alert', async () => {
+      const testElement2 = await createTestElement(html` <cds-alert-group status="info" style="width: 200px">
+        <cds-alert>
+          <pre style="overflow-x: auto">
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,</pre
+          >
```

Comment:
I think the `</pre>` tag should not be mangled/split like this. I'd but the entire closing tag on its own like like the opening tag.